examples/meta/src/kernel/sub_sequence_kernel.sg

```diff
@@ -0,0 +1,32 @@
+CSVFile f_feats_train("../../data/fm_train_dna.dat")
+CSVFile f_feats_test("../../data/fm_test_dna.dat")
+CSVFile f_labels_train("../../data/label_train_dna.dat")
+
+#![create_features]
+StringCharFeatures features_train(f_feats_train)
+StringCharFeatures features_test(f_feats_test)
+BinaryLabels labels_train(f_labels_train)
+#![create_features]
+
+#![create_appropriate_kernel]
+real maxlen = 1
```

Comment:
if this is called maxlen, this kind of is contrasting the math that says subsequences of exactly n